app/react/portainer/environments/wizard/index.ts

```diff
@@ -1,3 +1,4 @@
 export { HomeView } from './HomeView';
 export { EnvironmentTypeSelectView } from './EnvironmentTypeSelectView';
 export { EnvironmentCreationView } from './EnvironmentsCreationView';
+export { TLSFieldset } from './EnvironmentsCreationView/WizardDocker/APITab';
```

Comment:
I think this is time to move TLSFieldset to a different place. since you use it environments, so maybe in `app/react/portainer/environments/components`? if it's used in other places so `app/react/components`

Reply:
fixed